tests/run-pass/get_current_dir.rs

```diff
@@ -0,0 +1,6 @@
+// ignore-windows: TODO the windows hook is not done yet
+// compile-flags: -Zmiri-disable-isolation
+
+fn main() {
+    std::env::current_dir().unwrap();
```

Comment:
Best to name the test after the function, so `current_dir` in this case (or maybe `getcwd`).

But also, we actually have a non-negligible startup cost for every single test... so given that you grouped the implementation into `env`, and that the function is in the `env` module, it might make sense to test it together with the other `env` stuff as well?

Reply:
Oh actually you expanded and renamed this later so never mind.